docs/proguard-rules.txt

```diff
@@ -0,0 +1,39 @@
+-keep class com.microsoft.** { *; }
+
+## GSON 2.2.4 specific rules ##
+# Source: https://github.com/google/gson/blob/master/examples/android-proguard-example/proguard.cfg
+# Gson uses generic type information stored in a class file when working with fields. Proguard
+# removes such information by default, so configure it to keep all of it.
+-keepattributes Signature
+# For using GSON @Expose annotation
+-keepattributes *Annotation*
+-keepattributes EnclosingMethod
+# Gson specific classes
+-keep class sun.misc.Unsafe { *; }
+-keep class com.google.gson.stream.** { *; }
+# Gson specific classes used by Broker.
+-keep class * implements com.google.gson.TypeAdapterFactory
+-keep class * implements com.google.gson.JsonSerializer
+-keep class * implements com.google.gson.JsonDeserializer
+
+-dontnote sun.misc.Unsafe # see line 119
```

Comment:
not clear where the line 119 refers to, looks like a leftover from the original file.

Reply:
```suggestion
-dontnote sun.misc.Unsafe
```